packages/api-gw/src/adapters/index.js

```diff
@@ -1,7 +1,7 @@
-const healthyChecks = require('./healthyChecks');
-const link = require('./link');
+const healthyChecks = require('./healthyChecks/healthyChecks');
+const link = require('./link/link');
```

Comment:
Do we have a way to change the pattern that we are using in the `require` statements, just to avoid the duplication of the keywords?